Tasks/choco/Choco.ps1

```diff
@@ -62,28 +66,34 @@ function Install-Package
         [string] $IgnoreChecksums
     )
 
-    $expression = "$ChocoExePath install -y -f --acceptlicense --no-progress --stoponfirstfailure $Package --version $Version"
+    $expression = "$ChocoExePath install $Package"
+    
+    if ($Version){
+        $expression = "$expression --version $Version"
+    }
+
+    $expression = "$expression -y -f --acceptlicense --no-progress --stoponfirstfailure"
     
     if ($IgnoreChecksums -eq "true") {
         $expression = "$expression --ignorechecksums"
     }
 
-    Execute -Expression $expression
+    powershell.exe $expression
 }
 
 function Execute
 {
     [CmdletBinding()]
     param(
-        $Expression
+        $File
     )
 
     # Note we're calling powershell.exe directly, instead
     # of running Invoke-Expression, as suggested by
     # https://learn.microsoft.com/en-us/powershell/scripting/learn/deep-dives/avoid-using-invoke-expression?view=powershell-7.3
     # Note that this will run powershell.exe
     # even if the system has pwsh.exe.
-    $process = Start-Process powershell.exe -ArgumentList "-Command $Expression" -NoNewWindow -PassThru -Wait
+    $process = Start-Process powershell.exe -ArgumentList "-File $File -NoProfile -ExecutionPolicy Bypass"
```

Comment:
Can you re-add the -NoNewWindow -PassThru -Wait flags that `Start-Process` had before this PR?

Reply:
In addition to the current ones or as a replacement?